workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -668,11 +720,10 @@ $namespaces:
 hints:
 - class: 'sbg:maxNumberOfParallelInstances'
   value: 6
-- class: 'sbg:AWSInstanceType'
```

Comment:
why are we taking the instance type off this wf? is that defined in the tools/sub-wf?

Reply:
https://github.com/kids-first/kf-somatic-workflow/pull/63